### ISSUE TITLE
i18n and svelte

### ISSUE THREAD

**Issue by azmeuk** [nhn/tui.editor#1708]
Hi. I can successfully use tui.editor with svelte with a snippet like this:

```svelte
<script>
import { onMount } from "svelte"
import Editor from "@toast-ui/editor"
import "@toast-ui/editor/dist/toastui-editor.css"

let editor
let container

onMount(() => {
    editor = new Editor({
        el: container,
        ...
    });
});
</script>

<div id="editor" bind:this={container}></div>
```

However, when I am trying to use internationalization, rollup gives me this message:
```
./front/svelte/main.js → ./web/static/build/bundle.js...

[rollup-plugin-svelte] The following packages did not export their `package.json` file so we could not check the "svelte" field. If you had difficulties importing svelte components from a package, then please contact the author and ask them to export the package.json file.

- @toast-ui/editor

(!) Plugin node-resolve: Could not resolve import "@toast-ui/editor/dist/i18n/fr-fr" in /home/azmeuk/dev/yaal/lum1/front/svelte/tui/simple.svelte using exports defined in /home/azmeuk/dev/yaal/lum1/node_modules/@toast-ui/editor/package.json.
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
@toast-ui/editor/dist/i18n/fr-fr (imported by front/svelte/tui/simple.svelte)
created ./web/static/build/bundle.js in 15.6s
```

```svelte
<script>
import { onMount } from "svelte"
import Editor from "@toast-ui/editor"
import "@toast-ui/editor/dist/toastui-editor.css"
import "@toast-ui/editor/dist/i18n/fr-fr"

let editor
let container

onMount(() => {
    editor = new Editor({
        el: container,
        language: "fr",
        ...
    });
});
</script>

<div id="editor" bind:this={container}></div>
```

I am not sure what is wrong here, but I suppose a little tweak in package.json would enable full i18n support with svelte.

What do you think?

**Comment by js87zz** [comment]
@azmeuk 
I think it's probably the same reason for [this problem](https://github.com/nhn/tui.editor/issues/1677) with exports fields. This will be fixed in the next bug fix.

**Comment by js87zz** [comment]
@azmeuk 
The bug has been fixed in `v3.0.2`. Can you check the issue again?

**Comment by azmeuk** [comment]
It works with `v3.0.2`. Thank you!
